### ISSUE TITLE
[Question] the input file generated by CNVnator

### ISSUE THREAD

**Issue by QianXiaobo** [JH-Zhou/HandyCNV#12]
Hi Jinghang,

I am new in CNV analysis. I obtained the CNV results from CNVnator so that I should convert them to the standard format for HandyCNV. You mentioned that "The standard CNV input list should include the following columns (in order): Sample_ID, Chr, Start, End and CNV_Value". I am confused about "CNV_value", does it mean copy number in each CNV region? if yes, I will try to calculate copy number using -genotype in CNVnator as copy number. Or could you please give me any other suggest to convert the result of CNVnator to the standard format. 
I really appreciate your performance. Thanks a lot! 

Xb

**Comment by JH-Zhou** [comment]
Hi Xiaobo, 

Thank you for using this tool and asking. You correctly understand that "CNV_value" represents the copy numbers of the CNV region. However, this package can only works on SNP-based CNV results, because most functions are limited to the copy numbers from 0 to 4. Therefore, processing of the sequence-based CNV results is still not supported. And I have no experience in using CNVnator, so I'm sorry that I cannot help solve this problem. Thanks!

**Comment by QianXiaobo** [comment]
Jinghang, Thank you again!
You eliminated my confusion anyway. And I have an idea to preprocess my results. As we know, copy number (cn) =0,1 means deletion CNV and =3,4 means duplication CNV, (actually I don't know the difference between 0 and 1 (or 3 and 4)). With the results of cnvnator, I can know the type (dup or del) of each CNV, maybe I can transform the type of deletion to discrete number 0 or 1, and the same to duplication to 3 or 4. Therefore, HandyCNV can regard it as standard format input. I don't know whether it makes sense or not, Hah.

Have a nice holiday!
Xb

**Comment by JH-Zhou** [comment]
Yeah, it should work in that case, and it may be helpful to quick check the frequency of the CNVR and CNV-genes, have fun, hah
